actions/user_actions.jsx

```diff
@@ -422,10 +422,16 @@ export function autocompleteUsers(username) {
     };
 }
 
+export function setUserStatus(currentUserId) {
```

Comment:
Instead of adding a new action for this, could we have the StatusDropdown just use the action from mattermost-redux directly?

Reply:
Done.